bee/CMakeLists.txt

```diff
@@ -1,2 +1,3 @@
-add_subdirectory_ifdef(drivers)
+add_subdirectory(drivers)
+zephyr_include_directories(drivers)
```

Comment:
Is this line necessary?

Reply:
This include directory is for this build error：
![image](https://github.com/rtkconnectivity/hal_realtek/assets/153172365/f163d3f7-1a34-4b78-b5fe-1db2120cad07)